docs/wiki/configurator/failsafe-tab.md

```diff
@@ -4,8 +4,125 @@ sidebar_position: 5
+
+:::danger
+It is very important to set up your failsafe correctly to prevent your craft from flying away or crashing when you lose connection.
+:::
+
+![Failsafe tab](/img/betaflight_configurator_failsafe_tab.png)
+
+## Failsafe Stages
+
+There are two stages of failsafe:
+
+- **Stage 1** - entered when a flight channel has an invalid pulse length, the receiver reports failsafe mode or
+  there is no signal from the receiver at all, the channel fallback settings are applied to all channels and
+  a short amount of time is provided to allow for recovery.
+
+- **Stage 2** - entered when the error condition takes longer than the configured guard time while the craft is armed,
+  all channels will remain at the applied channel fallback setting unless overruled by the chosen procedure.
+
+:::note
+Prior to entering **stage 1**, channel fallback settings are also applied to individual AUX channels that have invalid pulses.
+:::
+
+## Valid Pulse Range Settings
+
+**Minimum length** - The minimum length of the valid pulse range. If a channel goes below this value, the FC will be put into failsafe mode.
+
+**Maximum length** - The maximum length of the valid pulse range. If a channel goes above this value, the FC will be put into failsafe mode.
+
+:::info
+For most cases, you should leave these settings at their default values.
+:::
+
+## Failsafe Switch
+
+This setting determines which stage of failsafe will be activated when manually engaging the Failsafe mode from your
+controller using the Aux Mode.
+
+- **Stage 1** - activates Stage 1 failsafe. This is useful if you want to simulate the exact signal loss failsafe behavior.
+- **Stage 2** - skips Stage 1 and activates the Stage 2 procedure immediately
+- **Kill** - disarms instantly (your craft will crash)
+
+## Stage 1 - Channel Fallback Settings
+
+These settings are applied to invalid individual AUX channels or to all channels when entering **stage 1**.
+
 :::note
-Sorry; this Wiki page has not yet been written.
+Values are saved in steps of 25usec, so small changes disappear
 :::
 
-For more information about Failsafe, see the [Wiki document](/docs/wiki/guides/current/GPS-Rescue-v4-5).
+You can set the channel fallback settings for each channel individually.
+
+The available options for each channel are:
+
+- **Auto** - Roll, Pitch and Yaw will be set to the center position, Throttle will be set to the minimum value.
+  (this is only available for the first 4 channels)
+- **Hold** - The channel will hold its last valid value.
+- **Set** - You can set a custom value for the channel.
+
+## Stage 2 - Failsafe Procedure
+
+There are three options for the failsafe procedure:
+
+- **Drop** (default) - The craft will disarm and fall out of the sky.
+- **Land** - The craft will attempt to land.
+- **GPS Rescue** - The craft will attempt to return to the home position.
+
+### Land options
+
+- **Throttle value used while landing** - The throttle value the craft will use while landing.
```

Comment:
The throttle value the craft will use while landing, unless the firmware includes Altitude Hold, in which case the throttle will be adjusted to provide a controlled descent.

Reply:
Oh wow, I didn't know this, is this something new in 4.6? Neat!